home.php

```diff
@@ -3,5 +3,23 @@
 echo "This is home page";
 // assign value
 $title = "Welcome to this website";
+# get menu
+$menu = "home_menu";
+
+function getMenuIndex($name){
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->